pages.nl/common/abd.md

```diff
@@ -0,0 +1,32 @@
+# adb
```

Comment:
The file has a typo in the name: should be `adb.md`.

Reply:
Good catch, fixed now :blush: